pkg/cmd/cli/backup/describe.go

```diff
@@ -78,7 +83,24 @@ func NewDescribeCommand(f client.Factory, use string) *cobra.Command {
 					fmt.Fprintf(os.Stderr, "error getting PodVolumeBackups for backup %s: %v\n", backup.Name, err)
 				}
 
-				s := output.DescribeBackup(&backup, deleteRequestList.Items, podVolumeBackupList.Items, details, veleroClient, insecureSkipTLSVerify, caCertFile)
+				var csiClient *snapshotv1beta1client.Clientset
+				// declare vscList up here since it may be empty and we'll pass the empty Items field into DescribeBackup
+				vscList := new(snapshotv1beta1api.VolumeSnapshotContentList)
+				if features.IsEnabled(velerov1api.CSIFeatureFlag) {
+					clientConfig, err := f.ClientConfig()
+					cmd.CheckError(err)
+
+					csiClient, err = snapshotv1beta1client.NewForConfig(clientConfig)
+					cmd.CheckError(err)
+
+					vscOpts := csi.NewListOptions(backup.Name)
```

Comment:
We can use the [`NewListOptionsForBackup`](https://github.com/vmware-tanzu/velero/blob/master/pkg/label/label.go#L59) function here.

Reply:
Yeah, I need to rebase. This was done before I made that commit.